src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -112,7 +112,7 @@ class ErrorHandler
      *
      * @return self The registered error handler
      */
-    public static function register(self $handler = null, $replace = true): self
+    public static function register(self $handler = null, bool $replace = true): self
```

Comment:
now that this class has been deprecated, all changes should be reverted (and the class should be removed actually)
see other classes too (should be easy to spot by rebasing)

Reply:
Reverted all changes in class. With deprecated you mean final? I couldn't find any others in the namespace.

Reply:
no, with deprecated, we mean deprecated.